src/game/client/gui/gamewindowtransitions.h

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file
+ *
+ * @author tomsons26
+ *
+ * @brief Classes for handling WND UI system transitions.
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#pragma once
+#include "always.h"
+#include "namekeygenerator.h"
+#include "subsysteminterface.h"
+#include <list>
+
+class GameWindow;
+class Transition;
```

Comment:
`GameWindow` and `Transition` classes are not used in this file (yet). Can be removed?